商品詳細ページ実装

nakahashi-yuki opened the pull request:
#what
商品の詳細ページへ行けるように実装。

#why
詳細ページに行くことにより、写真や商品名だけでなく、説明や細かな詳細情報を知ることができるようにするため。
また、出品者側は編集や削除。購入者側は購入手続きを行うためのページでもあるから。

nakahashi-yuki committed:
出品詳細画面の実装

nakahashi-yuki committed:
テーブルの外部キーを正しい記述に変更。

nakahashi-yuki committed:
rubocop修正

nakahashi-yuki commented:
購入者側からの詳細ページ　動画
https://gyazo.com/7f8c6694d5956c29995c473ad0bdb174

出品者側からの詳細ページ　動画
https://gyazo.com/decd2e0578b67b59cc93e73c70ded122

nakahashi-yuki committed:
Merge branch 'master' into 商品詳細ページ実装

Akinori0123 commented on app/views/items/show.html.erb:
```diff
@@ -16,54 +16,57 @@
+          <td class="detail-value"><%= @item.day.name %></td>
         </tr>
       </tbody>
     </table>
```
該当箇所にコメントができないためこちらに失礼いたします。
108行目にも@itemのカテゴリー名が表示されるようにしましょう。
```
<a href="#" class='another-item'><%= "商品のカテゴリー名" %>をもっと見る</a>
```

Akinori0123 commented on app/controllers/items_controller.rb:
```diff
@@ -3,6 +3,7 @@ class ItemsController < ApplicationController
 
```
該当箇所にコメントができないためこちらに失礼いたします。
```
（2行目）
before_action :authenticate_user!, except: [:index]
```
現在ログインしていないと商品の詳細ページにアクセスできない仕様になっておりませんでしょうか？
trelloにある通り、ログアウト状態でも商品詳細ページにアクセスできるようにする必要があります。
ご確認のほどお願いいたします。

Akinori0123 reviewed (commented):
レビュー依頼ありがとうございます。
二点コメントいたしましたのでご確認のほどお願いいたします。

nakahashi-yuki committed:
２点修正。

1.  before_action :authenticate_user!, except: [:index]
にshowアクション追加。

2.商品のカテゴリー名の表示を変更。

nakahashi-yuki committed:
Merge branch '商品詳細ページ実装' of https://github.com/nakahashi-yuki/furima-31022 into 商品詳細ページ実装

i-date reviewed (approved):
修正ありがとうございます！
修正点を拝見したところ問題ないことを確認できましたので、LGTMとさせていただきます。
以降の実装も引き続きこの調子で進めていきましょう。応援しております！
